src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <div class="product-card__container">
+        <div class="product-card__img">
+          <img
+            src="./images/imac.jpeg"
+            alt="i_mac"
+            class="card-img"
+          >
+        </div>
+        <div class="product-card__title">
+          <div class="title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </div>
+          <div class="product-card__id">
+            Код товара: 195434
+          </div>
+        </div>
+        <div class="product-card__rating">
+          <div class="rating">
+            <div class="stars stars--4">
+              <div class="stars__star"> </div>
+              <div class="stars__star"> </div>
+              <div class="stars__star"> </div>
+              <div class="stars__star"> </div>
+              <div class="stars__star"> </div>
```

Comment:
Я б назвав назви класів по іншому.
```suggestion
          <div class="rating">
            <div class="rating__stars stars--4">
              <div class="rating__star"> </div>
              <div class="rating__star"> </div>
              <div class="rating__star"> </div>
              <div class="rating__star"> </div>
              <div class="rating__star"> </div>
```

Reply:
Або взагалі не робити окремий блок вейтинг. А просто Вставити блок stars в елемент product-card__rating.